corehq/apps/userreports/tests/test_report_aggregation.py

```diff
@@ -861,7 +861,6 @@ def test_conditional_aggregation(self):
                 {
                     'type': 'conditional_aggregation',
                     'display': 'age_range',
-                    'field': 'age_at_registration',
```

Comment:
This wasn't actually from the cleanup script, directly, but it made me notice this error (`field` isn't relevant in this context).